### PR TITLE
fix array binding master key issue

### PR DESCRIPTION
**Pull request by SimbaGithub**
move cleanTempFiles to steam on data to check the status.

### CODE REVIEW

**Comment by sfc-gh-dszmolka** [comment]
another issue reported #374

**Comment by sfc-gh-dszmolka** [comment]
adding @sfc-gh-igarish as a reviewer here

File: lib/connection/bind_uploader.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const CREATE_STAGE_STMT = "CREATE OR REPLACE TEMPORARY STAGE "
  
 function BindUploader(options, services, connectionConfig, requestId)
 {
-	const MAX_BUFFER_SIZE = 1024 * 1024 * 100;
+	const MAX_BUFFER_SIZE = 1024 * 1024 * 1024 * 10;
```
Comment (sfc-gh-igarish) [inline lib/connection/bind_uploader.js:32]: You are increasing from 100 MB to 10 GB. Did you measure memory consumption?

**Comment by sfc-gh-igarish** [comment]
How about adding a test case?

**Comment by sfc-gh-dszmolka** [comment]
array binding master key issue is fixed with #419

**Pull request closed without merging**